ods_ci/libs/DataSciencePipelinesKfp.py

```diff
@@ -459,7 +459,9 @@ def create_run_from_pipeline_func(
         # the current_path will be ods-ci
         if current_path is None:
             current_path = os.getcwd()
-        my_source = self.import_souce_code(f"{current_path}/tests/Resources/Files/pipeline-samples/v2/{source_code}")
+        my_source = self.import_souce_code(
+            f"{current_path}/tests/Resources/Files/pipeline-samples/v2/cache-disabled/{source_code}"
```

Comment:
The default for KFP is `cache-enabled`. We should test the defaults first. Disabling the cache can be a parameter.

Reply:
I've addressed this comment as per our discussion in slack